controls/combobox/overview.md

```diff
@@ -12,23 +12,23 @@ The Telerik UI for .NET MAUI ComboBox allows users to select item/items from a d
 
 ![.NET MAUI ComboBox Overview](images/combobox-overview.png "ComboBox Overview")
 
-## Key Features
+## .NET MAUI ComboBox Key Features
 
-* [Editable and NonEditable mode]({%slug combobox-editmode-and-search%})—ComboBox supports both editable and noneditable state. When the control is in edit mode searching can be performed. 
-* [Searching Support]({%slug combobox-editmode-and-search%}) – ComboBox provides both case-sensitive and case-insensitive searching modes. The available options are: `Contains`, `StartsWith`, `ContainsCaseSensitive` and `StartsWithCaseSensitive`. 
-* [Single and Multiple Selection Support]({%slug combobox-selection%}) – ComboBox control has a support for single and multiple selection. You can specify the required selection using the `SelectionMode` property.
-* [Complex Object Support]({%slug combobox-databinding%}) – ComboBox control provides you a way to specify which property of your complex business object to be displayed in the control by setting `DisplayMemberPath`.
-* [Search Highlighting Text]({%slug combobox-editmode-and-search%}) – ComboBox control highlights the matching text inside the dropdown list based on the given input after searching is performed.
-* [Placeholder]({%slug combobox-configuration%}#placeholder) – The text which is used to give guidance to the end user on what can be entered/searched in the input. The watermark text is displayed when the input field is empty, or the selected item/s is/are cleared.
-* [Header and Footer]({%slug combobox-header-footer%}) – ComboBox for .NET MAUI gives you the ability to add header and footer inside the dropdown list. For this purpose, you will need to define the `HeaderTemplate` property and the `FooterTemplate` property. 
+* [Editable and NonEditable mode]({%slug combobox-editmode-and-search%})&mdash;.NET MAUI ComboBox supports both editable and noneditable state. When the control is in edit mode searching can be performed. 
```

Comment:
You don't need to replace the actual em dash (—) with the HTML code for an em dash. The result will be the same once the page is rendered on the screen. In general, there are two ways to render an em dash:
* Use the HTML code and accept its weird appearance in markdown.
* Use `Alt + 0151` which will generate an actual em dash (—) in the markdown markup.

I prefer the second option just because it is faster for me and looks prettier (to me), but you are free to choose your own approach.

Reply:
thanks for this note